Creating-Objects.md

```diff
@@ -28,10 +28,12 @@ When collecting user input, we often define the kind of input we expect using [[
 The schema would match a dictionary like this:
 
 ```json
-{"version": 34}
+{"length": 34.5}
 ```
 
-This schema describes a dictionary with a `version` key whose value is an `int`, and the `int` type is an object. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
+This schema describes a dictionary with a `length` key whose value is of type `Real`, and the `Real` type is one of our objects. We define these objects as classes in [`extensions/objects/models/objects.py`](https://github.com/oppia/oppia/blob/develop/extensions/objects/models/objects.py) and components in [`extensions.objects/templates`](https://github.com/oppia/oppia/blob/develop/extensions/objects/templates).
+
+Note that schemas can also use Python base types like `int`. These types are not objects, even though many of our objects are very similar to Python base types. For example, the `Real` object is essentially a `float`. However, `float` is not one of our objects.
```

Comment:
Schemas have their own definition for which keys are allowed for which types -- see schema_utils.py for details, or https://github.com/oppia/oppia/wiki/Schemas/8774d9fb027d82f39a6bbb80c8c08737615fa28d#write-a-schema

The 'type' key can only take a certain allowed set of values, not just any custom / builtin type.

Reply:
I removed this paragraph